dpypelines/pipeline/dataset_ingress_v1.py

```diff
@@ -37,6 +44,27 @@ def dataset_ingress_v1(files_dir: str, pipeline_config: dict):
         logger.error("Error occured while attempting to create notifier instance", err)
         raise err
 
+    try:
+        submitter_email = get_submitter_email()
```

Comment:
Could you add logger statements (logger.info and logger.error) in these new try/except blocks? One for when something in the try block is done successfully, another for if it fails and there's an error. See other existing try/excepts in v1_data_ingress.py for reference, it would be a couple of lines only.